R/extend_scenario_trajectory.R

```diff
@@ -134,24 +134,36 @@ extend_scenario_trajectory_old <- function(data,
 #'   year of the analysis.
 #' @param time_frame Numeric. A vector of length 1 indicating the number of
 #'   years for which forward looking production data is considered.
+#' @param baseline_scenario Character. A vector of length 1 indicating baseline
+#'   scenario
 #' @param target_scenario Character. A vector of length 1 indicating target
 #'   scenario
+#' @param emission_factors boolean. also summarise emmision factors? only needed
```

Comment:
for my understanding, are emmission factors only needed or only possible in the lrisk case? If it is the first I would suggest to also do it for trisk. It is not costly and saves some complication, a comment that it is only needed for lrisk would suffice

Reply:
removed distinction